### DISCUSSION TITLE
Clarity what "Server supports multi use" means

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#7255] [General]
"Using HTTP/2, server supports multi use" seems a little bit vague and bogus for non professionals, maybe we should change it to "Using HTTP/2, server supports **usage of connection for multiple times** ? This will be a much more clear message.

**Comment by jay** [comment]
How about "Using HTTP/2, server supports multiplexing"

**Reply by ghost** [reply]
That's a great idea, Jay!

**Reply by jay** [reply]
Thanks, landed in c6cac5a.
